app/question_list_helper.html

```diff
@@ -45,14 +45,13 @@ <h1 id="main_heading">My Questions</h1>
           </ul>
 
           <div id="panel_new" class="bb-tabpanel" role="tabpanel" aria-labelledby="tab1" aria-hidden="false">
-            <section id="myquestions" class="questions all" data-all="true"></section>
-            <button id="load-more" class="button-gray hide">Load more…</button>
+            <section id="myquestions" class="questions question-container all" data-all="true"></section>
```

Comment:
Sorry, forgot to say we should name this `js-question-container`. That conveys the idea that this class is only used for JS interactions and not styling.